db-manager/db_manager/__init__.py

```diff
@@ -0,0 +1,7 @@
+import logging
+
+logging.basicConfig(level=logging.INFO,

```

Comment:
The logging is already configured in the _main_ script (`db_manager/__main__.py`) and a setup / fixture for the tests (`tests/conftest.py`). Those places have it configured to log to the console (_INFO_ for main, _DEBUG_ for tests). Although the configuration here would be ignored if the logging is already configured, it is probably better not to have it in here.